karma.conf.js

```diff
@@ -46,6 +46,7 @@ module.exports = (karmaConfig) => {
     singleRun: true,
     preprocessors: {
       // do not include 'coverage' preprocessor for karma-coverage
+      // TODO
```

Comment:
I believe this will also be true for istanbul since the bundle is instrumented before it ever gets to Karma (the browser).

That said, an easy check would be to run tests and open `coverage/lcov-report/index.html` in browser an see if it looks right.